src/index.ts

```diff
@@ -29,10 +43,28 @@ export function runInContext(
   context: Context,
   options: Partial<IOptions> = {}
 ): Promise<Result> {
+  function getFirstLine(theProgram: Program) {
+    const firstLineOfProgram = theProgram.body[0] as ExpressionStatement
+    if (!!firstLineOfProgram) {
+      const firstLineExpression = firstLineOfProgram.expression as Literal
```

Comment:
Similar to above, I think you should explicitly check expression.type === 'Literal'